src/engraving/dom/edit.cpp

```diff
@@ -3484,6 +3484,8 @@ void Score::cmdDeleteSelection()
                         break;
                     }
                 }
+            } else if (e->isSoundFlag()) {
+                tick = toSoundFlag(e)->parentItem()->tick();
```

Comment:
It looks like we can use the SoundFlag tick directly: 

tick = toSoundFlag(e)->tick()